code/game/objects/effects/confetti.dm

```diff
@@ -0,0 +1,42 @@
+/obj/effect/effect/sparks/confetti
```

Comment:
This should probably be marked with a _vr, even if it is not an override it is useful to know what files are vorestation-specific and which are not.